integrations/standalone/tests/integration/dataclass-editor-entityclass.spec.ts

```diff
@@ -1,12 +1,25 @@
 import { expect, test } from '@playwright/test';
 import { DataClassEditor } from '../pageobjects/DataClassEditor';
 
-test('load data', async ({ page }) => {
-  const editor = await DataClassEditor.openDataClass(page, 'dataclasses/dataclass/EntityClass.d.json');
+let editor: DataClassEditor;
+
+test.beforeEach(async ({ page }) => {
+  editor = await DataClassEditor.openDataClass(page, 'dataclasses/dataclass/EntityClass.d.json');
+});
+
+test('master header', async () => {
+  await expect(editor.title).toHaveText('Entity Editor');
+});
+
+test('detail header', async () => {
+  await expect(editor.detail.title).toHaveText('Entity - EntityClass');
+});
```

Comment:
same here.
And by the way, sometimes it maybe makes more sense to test things as unit (vitest) instead with a full playwright test, especially if the test do not test anything with the backend only some frontend code...